db/seeds.d/02-roles_list.rb

```diff
@@ -53,5 +53,19 @@ def manage_organizations_permissions
     def view_permissions
       PermissionsList.permissions.select { |resource, name| name.start_with?('view_') }.map { |p| p.last.to_sym }
     end
+
+    def roles_descriptions
+      {
+        Role::MANAGER => 'Role granting all available permissions. With this role, user is able to do everything that admin can except for changing Settings.',
```

Comment:
Should the settings be Capitalized?

Reply:
I wanted to refer to menu item but you're probably right, will change it.